apps/api/src/persistence/persistence.module.ts

```diff
@@ -45,8 +46,18 @@ export class PersistenceModule {
         return {
             module: PersistenceModule,
             imports: [ConfigModule],
-            providers: [arangoConnectionProvider, repositoryProvider, idRepositoryProvider],
-            exports: [arangoConnectionProvider, repositoryProvider, idRepositoryProvider],
+            providers: [
+                arangoConnectionProvider,
+                repositoryProvider,
+                idRepositoryProvider,
+                DataExporter,
```

Comment:
Do we need this? We switched to manually instantiating the DataExporter instance in the CLI command constructor.

Reply:
removed